zivid_camera/src/zivid_camera.cpp

```diff
@@ -381,19 +428,14 @@ bool ZividCamera::isConnectedServiceHandler(IsConnected::Request&, IsConnected::
 void ZividCamera::publishFrame(Zivid::Frame&& frame)
 {
   const bool publish_points = points_publisher_.getNumSubscribers() > 0 || use_latched_publisher_for_points_;
-  const bool publish_color_img =
-      color_image_publisher_.getNumSubscribers() > 0 || use_latched_publisher_for_color_image_;
+  const bool publish_color_img = shouldPublishColorImg();
   const bool publish_depth_img =
```

Comment:
Suggest extracting to a function `shouldPublishDepthImage()` for consistency with `shouldPublishColorImg()`

Reply:
done